tsconfig.json

```diff
@@ -16,7 +16,6 @@
     "**/*.jsx",
     "**/*.cjs",
     "**/*.mjs",
-    ".next/types/**/*.ts",
-    "next.d.ts"
```

Comment:
we can remove the `next.d.ts` file as well, since the changes are now in Next.js

Reply:
6f7603d